acceptance/acceptance_test.go

```diff
@@ -454,19 +449,43 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 
 	printedRepls := false
 
+	// Compute repls for UUIDs. For each unique UUID we compute a unique replacement.
+	uuidRegex := regexp.MustCompile(`[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}`)
+	seen := map[string]struct{}{}
+	uuidRepls := []testdiff.Replacement{}
+	for _, relPath := range outputs {
+		s := testutil.ReadFile(t, filepath.Join(tmpDir, relPath))
+		matches := uuidRegex.FindAllString(s, -1)
+		for _, match := range matches {
+			if _, ok := seen[match]; ok {
+				continue
+			}
+
+			uuidRepls = append(uuidRepls, testdiff.Replacement{Old: regexp.MustCompile(match), New: fmt.Sprintf("[UUID-%d]", len(uuidRepls))})
+			seen[match] = struct{}{}
+		}
+	}
+
+	// Prepend UUID replacements to the final list of replacements so that they are applied frst
+	repls.Repls = append(uuidRepls, repls.Repls...)
```

Comment:
Why first? It can break other replacements, like username which is SP for example, which should happen before